app/mailers/card_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class CardMailer < ApplicationMailer
+  default from: "new-cards@flashcards.com"
+
+  def send_expired_cards(email)
+    @email = email
+    @url = "http://flashcards.com"
+
+    @cards = Card.created_before(Time.now)
+    mail(to: @email, subject: "Простроченные карточки на проверку от сайта Flashcards.com")    
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Trailing whitespace detected.